.github/workflows/windows-package.yml

```diff
@@ -29,3 +33,9 @@ jobs:
     - run: pacman -Ql mingw-w64-x86_64-openblas
     - run: pacman -Ql mingw-w64-x86_64-gcc-libgfortran
     - run: ls -l /mingw64/
+    - run: export PATH=cygpath "${pythonLocation}":$PATH
```

Comment:
looks like this does not work. Please change this to
`export PATH=${pythonLocation}:$PATH`